src/main/java/stream/flarebot/flarebot/commands/administrator/BanCommand.java

```diff
@@ -35,19 +36,23 @@ public void onCommand(User sender, TextChannel channel, Message message, String[
                     reason = MessageUtils.getMessage(args, 1);
                 FlareBotManager.getInstance().getAutoModConfig(channel.getGuild().getId())
                         .postToModLog(channel, user, sender, Punishment.EPunishment.BAN, reason);
-                channel.sendMessage(new EmbedBuilder()
-                        .setDescription("The ban hammer has been struck on " + user.getName() + " \uD83D\uDD28")
-                        .setImage(channel.getGuild().getId().equals(FlareBot.OFFICIAL_GUILD) ?
-                                "https://cdn.discordapp.com/attachments/226785954537406464/309414200344707084/logo-no-background.png" : null)
-                        .build()).queue();
-                channel.getGuild().getController().ban(channel.getGuild().getMember(user), 7 /*, reason*/).queue();
+                try {
+                    channel.getGuild().getController().ban(channel.getGuild().getMember(user), 7 /*, reason*/).queue();
+                    channel.sendMessage(new EmbedBuilder()
+                            .setDescription("The ban hammer has been struck on " + user.getName() + " \uD83D\uDD28")
+                            .setImage(channel.getGuild().getId().equals(FlareBot.OFFICIAL_GUILD) ?
```

Comment:
Oooooooooor you go ahead and use `attachments://filename.png` for the embed and upload the image. Oooooooooooooooor you could go ahead and set it to the URL from the website. And the guild ID has nothing to do with anything. What gave you that idea?